src/odemis/gui/cont/tabs/fastem_main_tab.py

```diff
@@ -205,6 +210,27 @@ def _toggle_user_settings_panel(self, _):
         )
         self.main_frame.Layout()
 
+    @call_in_wx_main
+    def _on_is_acquiring(self, mode):
+        """
+        Enable or disable the toolbar buttons depending on whether an acquisition
+        is already ongoing or not.
+
+        :param mode: (bool) whether the system is currently acquiring.
+        """
+        self.tab_data_model.tool.value = TOOL_NONE
+        self.tb.enable(not mode)
+
+    def _on_tool(self, selected_tool):
+        """Pause the SEM stream for a specific set of tools."""
+        if selected_tool in (TOOL_RECTANGLE, TOOL_ELLIPSE, TOOL_POLYGON):
+            if (
+                hasattr(self.tab_data_model, "semStream")
```

Comment:
Is there any reason that tab_data_model doesn't have semStream? How come the fastem (only) tab wouldn't have a SEM stream?